syntax = "proto3";

package order;

message OrderItem {
  int32 id = 1;
  int32 inventory_product_id = 2;
  int32 quantity = 3;
  string status = 4;
}